src/js/console.js

```diff
@@ -40,15 +56,25 @@ Console.prototype.error = function() {
   stderr(util.format.apply(this, arguments) + '\n');
 };
 
-var console = new Console();
+Console.prototype.time = function(label) {
+  this._times[String(label)] = process.hrtime();
+};
+
+Console.prototype.timeEnd = function(label) {
+  label = String(label);
 
-module.exports = {
-  log: console.log.bind(console),
-  debug: console.debug.bind(console),
-  info: console.info.bind(console),
-  warn: console.warn.bind(console),
-  error: console.error.bind(console),
-  Console: Console,
-  _stdout: stdout,
-  _stderr: stderr,
+  var time = this._times[label];
+  if (time == null) {
```

Comment:
strict equal?

Reply:
`variable == null` is both true if variable is `null` or `undefined`. I think this shall be one exception for rule of strict equal.

Reply:
See https://github.com/airbnb/javascript#comparison--eqeqeq, we should follow this lint rule.